tools/scripts/build/build-external-curricula-data.ts

```diff
@@ -39,17 +39,22 @@ export const orderedSuperBlockInfo = [
   { dashedName: SuperBlocks.MachineLearningPy, public: true },
   { dashedName: SuperBlocks.RespWebDesign, public: true },
   { dashedName: SuperBlocks.PythonForEverybody, public: true },
-  { dashedName: SuperBlocks.JsAlgoDataStruct, public: false },
+  { dashedName: SuperBlocks.TheOdinProject, public: true },
```

Comment:
Should be false for now, we can set it to true after we release it.
```suggestion
  { dashedName: SuperBlocks.TheOdinProject, public: false },
```

Reply:
Nice catch. I mixed up the order of release 😅